### PR TITLE
Fixed full-text search (do not show expired record)#138

### PR DESCRIPTION
**Pull request by prazuma**

### CODE REVIEW

File: app/full_text_search.py [hunk 1]
```diff
@@ -51,7 +58,9 @@ def search(repo, query_txt, max_results):
         query_string=query_txt, options=options))
     for document in index_results:
         id = document.fields[0].value
-        results.append(model.Person.get_by_key_name(repo + ':' + id))
+        result = get_record_from_datastore(repo, id)
```
Comment (gimite) [inline app/full_text_search.py]: Use Person.get() instead:
https://github.com/google/personfinder/blob/master/app/model.py#L209

Person.get() also has feature to check if it's expired.

File: app/full_text_search.py [hunk 2]
```diff
@@ -51,7 +58,9 @@ def search(repo, query_txt, max_results):
...
+        if result:
+            results.append(get_record_from_datastore(repo, id))
```
Comment (gimite) [inline app/full_text_search.py]: Reuse "result" instead of calling get_record_from_datastore() twice.

Comment (prazuma) [inline app/full_text_search.py]: done

File: app/full_text_search.py (hunk 1 above)
Comment (prazuma) [inline app/full_text_search.py]: done

**Comment by gimite** [comment]
LGTM. Merged.

**Comment by prazuma** [comment]
(∩´∀｀)∩ﾜｰｲ

**Pull request merged**